notifications/src/main/kotlin/notifications/service/EmailNotificationService.kt

```diff
@@ -0,0 +1,66 @@
+package notifications.service
+
+import com.github.kittinunf.result.Result
+import com.github.kittinunf.result.flatMap
+import mu.KLogging
+import notifications.smtp.SMTPService
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Component
+import provider.D3ClientProvider
+
+private const val NOTIFICATION_EMAIL = "noreply@d3.com"
+const val D3_WITHDRAWAL_EMAIL_SUBJECT = "D3 withdrawal"
```

Comment:
I think it is common practice just to use the same subject name for all notifications. Smth like `D3 notification` should be fine